x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ml/inference/TrainedModelDefinitionTests.java

```diff
@@ -302,4 +306,67 @@ public void testRamUsageEstimation() {
         assertThat(test.ramBytesUsed(), greaterThan(0L));
     }
 
+    public void testMultiClassIrisInference() throws IOException {
+        // Fairly simple, random forest classification model built to fit in our format
+        // Trained on the well known Iris dataset
+        String compressedDef = "H4sIAMHqMV4C/+1b7Y6bOhB9lVV+b5HHX9h9g/sMVYVo4iRIbBIBaXtV9d0vJr23uzGKJwwsu" +
```

Comment:
Where does this string come from? Could we have a resource file with plain text instead?

Reply:
Its a model, the comment says as much. Writing a multi-class model by hand that adequately test paths in a semi-non-trivial way was becoming frustrating. 

This model was specifically trained in scikit-learn and transformed into our format. 

This is the only test that utilizes this string. It is not out of the ordinary to have tests access an immutable string.